docs/tutorials/building-your-first-bot.mdx

```diff
@@ -0,0 +1,342 @@
+}
+```
+
+That's it! The bot will now be able to see the current and max
+health of the enemy, and whether or not the player is attacking. See the full [state documentation](/integrating-with-unity/defining-states)
+to see all the ways you can define state within your game, including grabbing all of the state from a MonoBehaviour.
+
+## Implement actions your bot can take
+
+Similar to states, we will now tag the actions that a bot can take, using [`[RGAction]`](/integrating-with-unity/defining-actions). In our case, a bot can either move or attack.
+
+Open the `Assets/FirstBotDemo/Demo/Scripts/Player/PlayerInputControl.cs` file, and replace the contents with the following.
+This change does a few things:
+
+1. Adds an editor field that allows us to mark a specific prefab instance as being a bot (vs a character controlled by a real player).
+2. Uses that field to ensure that keyboard/input handling only applies to non-bot game objects.
+3. Marks the movement method as being an action that the bot can perform.
+
+```csharp
+using RegressionGames;
+using UnityEngine;
+
+namespace FirstBotDemo
+{
+    public class PlayerInputControl : MonoBehaviour
+    {
+        [Header("Character Input Values")]
+        public Vector2 move;
+        public Vector2 look;
+        public bool jump;
+
+        [Header("Movement Settings")]
+        public bool analogMovement;
+
+        [Header("BotSettings")] public bool isBot = true;
+
+        private void Update()
+        {
+            if (!isBot)
+            {
+                MoveInput(new Vector2(Input.GetAxis("Horizontal"), Input.GetAxis("Vertical")));
+                JumpInput(Input.GetButtonDown("Jump"));
+            }
+        }
+        
+        [RGAction("MoveInDirection")]
+        public void MoveInput(Vector2 newMoveDirection)
+        {
+            move = newMoveDirection;
+        }
+        
+        public void JumpInput(bool newJumpState)
+        {
+            jump = newJumpState;
+        }
+    }
+}
+```
+
+Then, open the `Assets/FirstBotDemo/Demo/Scripts/Player/PlayerAttack.cs` file again, and add the following method to the class,
+which defines an action that allows a bot to target an enemy and use an ability:
+
+```csharp
+[RGAction]
+public void SelectAndAttackEnemy(int enemyId, int ability)
+{
+    var enemy = RGFindUtils.Instance.FindOneByInstanceId<EnemyController>(enemyId);
+    SelectEnemy(enemy);
+    Attack(abilities[ability]);
+}
+```
+
+
+As a last step here, in the player input code, you will see that we introduced an `isBot` field to the `PlayerInputControl` component.
+This will allow our player to be controlled by the bot logic when it is set to true. However, there is an existing `Player`
+GameObject in the scene, which we want to control using our manual input controls. Click this `Player` in the object hierarchy,
+find the `Player Input Control` component, set `Is Bot` to unchecked, and then save (you may find that it is already unchecked).
```

Comment:
Since we're doing all of this in the context of bots rather than human players, I think it makes more sense to default `isBot` to `true` and have the user check the box on their bot prefab. Once you started talking about the `Player` object, I was confused about how the Player is used in this tutorial, but it looks like they aren't - this step is just to make sure the Player doesn't "break" from these changes

Reply:
Maybe I can remove all of this because from all of my tests so far, no change is actually needed. It seems like the defaults work as expected.